### DISCUSSION TITLE
routes for 2 collections

### DISCUSSION THREAD

**Discussion by mcamprodon** [statamic/cms#6878] [Q&A]
Hello! I have a routing question. I have 2 collections:
- repors
- articles
Each repor have multiple articles
I want to create routes for articles like {slug_repor}/{slug_article}
I need help to create it. With the same collection this is easy with {parent_uri} but with 2 collections I don't know how to create it.

**Comment by krokusmoen** [comment]
Having the same issue.

Have 3 collections:

- Pages
- Destinations
- Products

A "Product" is attached to one entry in "Destinations".

The wanted route on the "Product" collection would be "/{destination:slug/{slug}". Is that achievable?

**Comment by edalzell** [comment]
You'll both need custom routes.
